processor/src/main/java/org/mapstruct/ap/internal/prism/MappingInheritanceStrategyPrism.java

```diff
@@ -25,6 +25,32 @@
  * @author Andreas Gudian
  */
 public enum MappingInheritanceStrategyPrism {
```

Comment:
Do we need a unit test to make sure that the parameters here are correct? I know they would break the other tests. I can add it later on as well